src/test/java/reposense/authorship/FileInfoExtractorTest.java

```diff
@@ -142,7 +142,7 @@ public void generateFileInfo_fileWithoutSpecialCharacters_correctFileInfoGenerat
     }
 
     @Test
-    public void getNonBinaryFilesList_directoryWithBinaryFiles_success() {
+    public void getFilesList_directoryWithBinaryFiles_success() {
```

Comment:
need to add another test for non-binary files

Reply:
Actually this test already contains tests for both binary and non-binary files already. I will change the name of the test.